builder.config.json

```diff
@@ -85,105 +85,122 @@
         "type": "ConsoleIO.Print",
         "is_interactive": false,
         "is_branching": false,
-        "category":  0
+        "category":  0,
+        "menu_category": 5
```

Comment:
Hi @safydy, how hard would it be to just use `category` instead of adding `menu_category`? Category was originally added for this purpose and it's no longer used for anything anyhow 😛